_pages/webextensions.md

```diff
@@ -12,160 +12,173 @@ parent: activities
 
-If you enjoyed creating your own add-on, the [dev-addons@mozilla.org mailing list](https://mail.mozilla.org/listinfo/dev-addons), IRC channels #addons, #webextensions and #extdev, Telegram group [@addonschat](https://telegram.me/addonschat), and [Add-ons Discourse Forum](https://discourse.mozilla.org/c/add-ons) are full of people just like you! Join these channels to ask questions, brainstorm ideas, and stay informed.
+🔍  [Let’s Write a Web Extension by Blake Winton](https://hacks.mozilla.org/2015/09/lets_write_a_webextension/)
 
-Also, keep your eyes on the [Add-ons Blog](https://blog.mozilla.org/addons/) to learn more about what’s going on in the world of add-ons.
+🔍  [Create a Simple WebExtension by David Wals](https://davidwalsh.name/web-extensions)
 
-__Are you interested in exploring other ways to contribute to the add-ons community?__
+👍  [Browser Compatibility Table for APIs](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Browser_support_for_JavaScript_APIs)
 
-Visit our [wiki](https://wiki.mozilla.org/Add-ons/Contribute) to learn more.
 
-Thanks again! We look forward to seeing you next time.
+## Event Follow-Up 
 
-## Output
+* Immediately after every event, share the link to the impact form with your attendees: [mzl.la/howwasit](mzl.la/howwasit). While you’re at it, why not also send a copy of your slide deck and invite the attendees to join the Telegram group [@addonchats](https://t.me/addonschat)? 
 
 * Fill out the post-event metrics on the event page you set up on the Reps portal.
-* Tell us about your event. This can be before or after. We love to hear about upcoming plans, and of course love to hear about what was achieved at the event with a picture or two!
-  * The Discourse thread can be found [here](https://discourse.mozilla.org/t/activate-build-your-own-webextension-add-on-for-firefox/13866)
 
-## Tutorials & Other Resources
+* [Tell us about your event](https://discourse.mozilla.org/t/activate-build-your-own-webextension-add-on-for-firefox/13866) on Discourse. We love to hear about upcoming plans, and of course love to hear about what was achieved at the event with a picture or two!
 
-__Tutorials__
 
-* [Your first WebExtension](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Your_first_WebExtension)
-* [Let’s Write a Web Extension by Blake Winton](https://hacks.mozilla.org/2015/09/lets_write_a_webextension/)
-* [WebExtensions: An Example Add-on Repository with Test Harnesses by standard8](https://blog.mozilla.org/standard8/2016/11/11/webextensions-an-example-add-on-repository-with-test-harnesses/)
-* [Create a Simple WebExtension by David Walsh](https://davidwalsh.name/web-extensions)
-* [Building Firefox Add-ons with Web Extensions by Viswaprasath Ks](http://www.slideshare.net/VISWAPRASATHKS)
+## Acknowledgements
+Thank you to Viswaprasanth Ks, Santosh Viswanatham, Trishul Goel, Karthic Keyan, Andre Garzia Daniele Scasciafratte, and Michael Kohler.
```

Comment:
not sure why exactly, I'm just reviewing stuff ;) but thanks!

nit: there is a comma missing before Daniele

Reply:
I appreciate your attention to detail. :) Yes, let's get that comma in there.